.github/workflows/integration-tests.yml

```diff
@@ -37,4 +36,4 @@ jobs:
           make install
       - name: Run tests
         run: |
-          go test -tags "integration,${{ matrix.suite }}" -timeout 20m -parallel 1 -v -shuffle=on ./pkg/integration/...
```

Comment:
Why this change? At face value, I'd read it as an indication of the test suite getting worse.

Reply:
Agreed, but we're using 1.16 for now, and that option was added with go 1.17, it's messing up with the tests,
I added a comment to clarify this.